firmware/sys/network/include/cpumac.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @ingroup     
+ * @{
+ * @file
+ * @brief       cpumac address
+ *
+ * @author      RocioRojas <rociorojas391@gmail.com>
+ *
+ */
+#ifndef CPUMAC_H
+#define CPUMAC_H
+
+#include "periph/cpuid.h"
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+/**
+ * @brief  This function get cpuid address
+ * @param [in] address mac address of device
+ * @param [out] id The serial number of the CPU of length CPUID_LEN (should be defined in the
+ * CPU's cpu_conf.h)
+ */
+void cpumac(char *input, char *output);
```

Comment:
function's name is not descriptive,  `get_cpuid` or something like that sounds better